classes/Report/Report.php

```diff
@@ -173,15 +173,15 @@ public function save(string $real_fname)
             $st->bindValue(4, $ct->format('Y-m-d H:i:s'), \PDO::PARAM_STR);
             $st->bindValue(5, $this->data['org'], \PDO::PARAM_STR);
             $st->bindValue(6, $this->data['external_id'], \PDO::PARAM_STR);
-            $st->bindValue(7, $this->data['email'], \PDO::PARAM_STR);
-            $st->bindValue(8, $this->data['extra_contact_info'], \PDO::PARAM_STR);
+            $st->bindValue(7, isset($this->data['email']) ? $this->data['email'] : null, \PDO::PARAM_STR);
```

Comment:
Hm... Why not use `$this->data['email'] ?? null` instead? It's a little shorter.